platform/wab/src/wab/client/components/sidebar/token-controls.tsx

```diff
@@ -273,12 +284,25 @@ export const TokenRow = observer(function TokenRow(props: {
 
 export const TokenFolderRow = observer(function TokenFolderRow(props: {
   name: string;
+  normalPath?: string;
+  tokenType: TokenType;
   matcher: Matcher;
   groupSize: number;
   indentMultiplier: number;
   isOpen: boolean;
 }) {
-  const { name, matcher, groupSize, indentMultiplier, isOpen } = props;
+  const {
+    name,
+    normalPath,
+    tokenType,
+    matcher,
+    groupSize,
+    indentMultiplier,
+    isOpen,
+  } = props;
+  const tokenControls = useTokenControls();
+  const studioCtx = useStudioCtx();
```

Comment:
Not sure about useStudioCtx here, it seems to break hot reload (error: can't access context before initialization)

Reply:
Let's keep it this way, not worth the issue if the only broken thing is the hot reload (which is already kinda broken anyways)